pkg/detectors/phraseaccesstoken/phraseaccesstoken_test.go

```diff
@@ -0,0 +1,115 @@
+package phraseaccesstoken
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/engine/ahocorasick"
+)
+
+var (
+	validPattern   = "1a2b3c4d5e6f7890abcdef1234567890abcdef1234567890abcdef1234567890"
+	invalidPattern = "7cf4135a4e7f7ac228d36f210f151917a86f5dbd6"
+	keyword        = "phrase"
+)
```

Comment:
Can we use more than one valid pattern? 
Also, consider using the secret directly in the input text instead of assigning it to a variable; it'll improve readability.